packages/venia-concept/src/components/App/appWrapper.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useWindowSize } from '@magento/peregrine/src/hooks/useWindowSize';
+import App from './container';
+
+import AppContext from './context';
+
+/**
+ * A functional wrapper so that we can provide context to the entire app
+ * without having to rewrite the App file itself.
+ */
+function AppWrapper() {
```

Comment:
I wrote this temporarily since I didn't want to have to refactor `index` or `App` to be a functional component. App is close but it uses `getDerivedStateFromError` which is [not yet hook-ified](https://reactjs.org/docs/hooks-faq.html#how-do-lifecycle-methods-correspond-to-hooks):

> getDerivedStateFromError: There are no Hook equivalents for these methods yet, but they will be added soon.